sql/moz-fx-data-shared-prod/subscription_platform_derived/apple_subscriptions_v1/metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: Apple Subscriptions
+description: >
+  Apple subscriptions.
+owners:
+  - srose@mozilla.com
+labels:
+  schedule: daily
```

Comment:
should this be labelled as incremental?

Reply:
No, it rebuilds the entire table every time.  I considered trying to make these incremental, but since previously they were views that got re-run every time they were used, doing full table rebuilds seems the simplest approach with the least chance to accidentally introduce bugs/edge cases.